core/src/test/java/org/apache/any23/extractor/yaml/YAMLExtractorTest.java

```diff
@@ -93,4 +92,17 @@ public void nullTest()
         RepositoryResult<Statement> docs = getStatements(null, null, RDF.NIL);
         Assert.assertTrue(Iterations.asList(docs).size() == 2);
     }
+
+    /**
+     * Comma separated values are parsed as well.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void csvTest()
+            throws Exception {
+        assertExtract("/org/apache/any23/extractor/csv/test-comma.csv");
```

Comment:
I am not familiar with YAML, but is CSV (RFC 4180 Section 2 and all other variations) a strict subset of YAML or is it incidental that this test file works with the YAML parser? If it is only incidental we shouldn't be testing it here.